lessons/en/basics/date_time.md

```diff
@@ -76,7 +76,7 @@ iex(15)> NaiveDateTime.utc_now
 ~N[2029-01-21 19:55:10.008965]
 ```
 
-But it has both the current time and date, so you can play with adding time, for example:
+But it has both the time and date, so you can play with adding time, for example:
```

Comment:
The same as above — `NaiveDateTime` doesn't have to contain only the _current_ time.